app/styles/components/post-view.scss

```diff
@@ -0,0 +1,66 @@
+.post-view {
+  font-family: serif;
+  font-size: 16px;
+  @include margin-top-xl();
+
+  .post-container {
+    @include padding-bottom-xl();
+    @include padding-top-xl();
+    background-color: $gray-extralight;
+  }
+
+  h1 {
+    @include display-font();
+    font-size: 26px;
+  }
+
+  blockquote {
+    @include display-font();
+    @include padding-bottom-l();
+    @include padding-top-l();
+    border-left-style: none;
+    font-size: 20px;
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  header {
+    @include padding-bottom-l();
+  }
+
+  .banner {
+    img {
+      width: 100%;
+    }
+  }
+
+  footer {
+    @include margin-top-l();
+    @include padding-top-xl();
+    @include padding-bottom-xl();
+    font-size: 12px;
```

Comment:
Properties should be ordered background-color, font-family, font-size, text-align